Animations/src/core/Application.cpp

```diff
@@ -96,7 +98,7 @@ namespace MathAnim
 			//globalThreadPool = new GlobalThreadPool(true);
 
 			// Initiaize GLFW/Glad
-			window = new Window(1920, 1080, winTitle, WindowFlags::OpenMaximized);
+			window = new Window(1920, 1080, winTitle, WindowFlags::None);
```

Comment:
I understand you may have changed some settings to personal preference, but until I add a preferences manager I'd like to keep this as `::OpenMaximized` for now. You can definitely leave it in local copy though 👍